packages/babel-plugin-minify-dead-code-elimination/src/index.js

```diff
@@ -3,6 +3,15 @@
 const some = require("lodash.some");
 const { markEvalScopes, isMarked: isEvalScopesMarked, hasEval } = require("babel-helper-mark-eval-scopes");
 
+function forEachPrevSibling(path, callback) {
```

Comment:
Considering that DCE has a lot of cases of `path.parentPath.getSibling(path.parentPath.key - 1)` and `path.parentPath.getSibling(path.parentPath.key + 1)`, I think it makes sense to add helper traversal methods to Babel.